main.go

```diff
@@ -1284,8 +1284,27 @@ NBITS defaults to 2048.
 
 	r.Dispatch("rekey", &Help{
 		Summary: "Re-key your Vault with new unseal keys",
-		Usage:   "safe rekey [OPTIONS]",
+		Usage:   "safe rekey [--gpg email@address ...] [--num-unseal-keys <int>] [--keys-to-unseal <int>]",
 		Type:    DestructiveCommand,
+		Description: `
+Rekeys Vault with new unseal keys. This will require a quorum
+of existing unseal keys to accomplish. This command can be used
+to change the nubmer of unseal keys being generated via
```

Comment:
number* :)
Also, can we specify the defaults for not giving the numbers?

Reply:
👍